apache/Dockerfile

```diff
@@ -9,10 +9,18 @@ RUN pecl install apcu \
     && pecl install xdebug
 
 RUN apt-get update \
-    && apt-get install -y cron zlib1g-dev # dependency for php zip extension
+    && apt-get install -y cron zlib1g-dev \
+            libfreetype6-dev \
```

Comment:
Nope. \ stands for "line continuation". libfreetype6-dev is yet another package installed using apt-get install on the line above.

(I tested it, it works)